cvapipe_analysis/tools/cluster.py

```diff
@@ -149,35 +149,35 @@ def distribute_by_blocks(self):
 
 class FeaturesDistributor(Distributor):
     def __init__(self, control):
-        super().__init__(control)
+        super().__init__(control, 'computefeatures')
```

Comment:
Instead of hardcoding the step name, you can get it with `self.step_name`. Can you try that and see if it works?

Reply:
I didn't know that was accessible, thanks! done.